app/controllers/katello/api/rhsm/candlepin_dynflow_proxy_controller.rb

```diff
@@ -0,0 +1,46 @@
+module Katello
+  class Api::Rhsm::CandlepinDynflowProxyController < Api::V2::ApiController
```

Comment:
Part preparing for the future, part for the sake of clarity.

In rails 5, parameter parsing is done by base controller, so we will need a different inheritance for upload_packages action.
In rails 4, as I found out later, the parsing is done by rack middleware, so I don't have to inherit `Metal` directly.